### DISCUSSION TITLE
How to use it?

### DISCUSSION THREAD

**Discussion by VityaSchel** [i18next/next-i18next#1826] [Q&A]
I don't understand it at all. There is just one example that describes very specific use-case. Despite having 3.2k stars and recommendation from next.js this repository only have one maintainer and poor documentation. After reading README.md and website I still can't get it to work. I tried replicating the basic example, but it asks for "fs" module and I see "Module not found: Can't resolve 'fs'" error. I wanted to change locales path location, but ended up using "public" folder because there is simply no information on how to use localePath, I'm not even sure what format does it need, why path.resolve needed, should it have trailing slash and what folder structure is acceptable. How to change folder structure, if it's possible or not. Can I have en.json instead of en/common.json?

**Comment by kachkaev** [comment]
Please contribute with a PR if you see scope for specific improvements. You can find plenty of open-source projects using `next-i18next` here: https://github.com/isaachinman/next-i18next/network/dependents?package_id=UGFja2FnZS0yMjQxMDM4Mjg%3D
